### PR TITLE
Jetpack connect: Change copy of 'searching for available upgrades'

### PR DESCRIPTION
**Pull request by johnHackworth**
"Searching Available Upgrades" wasn't a good description on what we were doing in that step (pretty much: refreshing sites info so we can show the plans for the newly connected site), so we are changing it for a more generic "finishing up"
image

How to test
========
1. Go to http://calypso.localhost:3000/jetpack/connect/ and connect a site that is running latest jetpack's master
2. Once the site is authorized, you should see the "finishing up" message for some seconds

### CODE REVIEW

File: client/signup/jetpack-connect/authorize-form.jsx [hunk 1]
```diff
@@ -249,7 +249,7 @@ const LoggedInForm = React.createClass( {
 		}
 
 		if ( authorizeSuccess ) {
-			return this.translate( 'Searching Available Upgrades' );
+			return this.translate( 'Finishing up' );
```
Comment (mtias) [inline client/signup/jetpack-connect/authorize-form.jsx]: Maybe add some context here for translation?

**Comment by rickybanister** [comment]
Other than @mtias's suggestion, this looks good. Might be worth throwing an `!` in there for fun.

**Comment by johnHackworth** [comment]
process ok, added context and a `!`, @rickybanister @mtias

**Comment by rickybanister** [comment]
Let's get this merged!

**Pull request merged**